test/parallel/test-message-port.js

```diff
@@ -18,6 +18,21 @@ const { MessageChannel, MessagePort } = require('worker_threads');
   }));
 }
 
+{
+  const { port1, port2 } = new MessageChannel();
+
+  port1.on('message', common.mustCall((message) => {
+    assert.strictEqual(message, 4);
```

Comment:
Could `port.onmessage =` also be tested?

Reply:
Used it [here (line 31)](https://github.com/itaysabato/node/blob/54974ae3a2028b2f38d8c363dee54937349ab4c7/test/parallel/test-message-port.js#L31).
Did you have something more elaborate in mind?

Reply:
I commented on the wrong line, sorry. My suggestion is to use it for both port1 and port2 (also do `port1.onmessage = ...`

Reply:
Sure, why not :)